torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -3521,17 +3518,22 @@ PyTorchModelLoader::PyTorchModelLoader(
         glow::strFormat("Number of Graph inputs %lu must match the "
                         "number of provided inputs %lu.",
                         graphInputValues.size(), inputs.size()));
-
     // Create Glow Placeholders for inputs.
     for (size_t i = 0; i < graphInputValues.size(); ++i) {
       const torch::jit::Value *inputValue = graphInputValues[i];
       glow::Placeholder *ph;
       if (!inputMeta.empty()) {
         if (inputValue->type()->kind() == c10::TypeKind::TensorType) {
-          glow::Type t(scalarTypeToElemKind(inputMeta[i].type),
-                       inputMeta[i].dims);
+          glow::ElemKind elemKind;
+          if (inputMeta[i].type != at::kQUInt8) {
+            elemKind = scalarTypeToElemKind(inputMeta[i].type);
+          } else {
+            elemKind = ElemKind::Int8QTy;
```

Comment:
Why this?

Reply:
Because we dont have rescale anymore and therefore we need all our placeholder change all  uint8 to int8 in type, and latter change all uint8 to int8 in data during runtime.